### ISSUE TITLE
Support git show

### ISSUE THREAD

**Issue by hexium310** [chitoku-k/fzf-zsh-completions#95]
Do any patterns of the arguments other than the following exist?

```sh
# Display the commits, the branches and the tags (_fzf_complete_git-commits)
$ git show **

# Display the changed files of current commit that is displayed by `git show --pretty=format: --name-only <rev>`
$ git show [<rev>] -- **

# Display the files in current commit (_fzf_complete_git-ls-tree)
$ git show <rev>:**

# Display nothing
$ git show <rev>:<path> -- **
```

https://git-scm.com/docs/git-show

**Comment by hexium310** [comment]
```sh
$ git show <rev> [<rev>...] <path> [<path>...] **
```

**Comment by chitoku-k** [comment]
For the following cases, we can probably force the user input to be delimited by `--` as in another subcommands:

```zsh
$ git show <rev> <rev> ** # => Display <rev>
$ git show <rev> <rev> -- ** # => Display <path>
```

I am not sure about the following. Needs more context...

```zsh
$ git show [<rev>:<path>...] rev1 [<rev>:<path>...] [--] ** # => Display <path> contained in `rev1`
```

**Comment by hexium310** [comment]
> I am not sure about the following. Needs more context...
> 
> ```shell
> $ git show [<rev>:<path>...] rev1 [<rev>:<path>...] [--] ** # => Display <path> contained in `rev1`
> ```

For example, when the command is below, the changed files in feat/git-show should be displayed. In other words, we need to get 2nd or later arguments.
```sh
$ git show master:LICENSE feat/git-show master:.editorconfig -- **
```

**Comment by chitoku-k** [comment]
Could you elaborate on what is the exact problem?

**Comment by hexium310** [comment]
It is that `_fzf_complete_git_parse_argument` get only 1st argument.

**Comment by chitoku-k** [comment]
No, its first argument should specify the index of arguments to retrieve.

**Comment by hexium310** [comment]
Sorry, it is not to get but to return.
```sh
$ arguments='git show master another-branch 3rd-branch'
$ _fzf_complete_git_parse_argument $arguments 1 '' ''
master
```
If it become to return master, another-branch and 3rd-branch, the problem of `git show [<rev>:<path>...] rev1 [<rev>:<path>...] [--] **` will be resolved.

**Comment by chitoku-k** [comment]
```
_fzf_complete_git_parse_argument 2 $arguments ''
```
results in `another-branch` in that case.

**Comment by chitoku-k** [comment]
Closed by #97
